R/tokens_ngrams.R

```diff
@@ -48,8 +48,7 @@ tokens_ngrams.default <- function(x, n = 2L, skip = 0L, concatenator = "_") {
 ## the grammatical rules of quanteda (inputs character, outputs tokens),
 ## but starts with "tokens_"
 #' @importFrom stats complete.cases
-tokens_ngrams.character <- function(x, n = 2L, skip = 0L, concatenator = "_") {
-
+tokens_ngrams_character <- function(x, n = 2L, skip = 0L, concatenator = "_") {
```

Comment:
I changed this to not be a method of the generic for which it was inappropriately named, but I think we agree that for 4.0, we should remove this anyway since it shortcuts the tokenisation step.